darts/tests/models/forecasting/test_local_forecasting_models.py

```diff
@@ -84,6 +85,7 @@
     (NaiveSeasonal(), 32),
     (NaiveMean(), 37),
     (NaiveDrift(), 39),
+    (NaiveMovingAverage(W=5), 34),
```

Comment:
Shouldn't this be `input_chunk_length`? How come this test is passing?

Reply:
It wasn't actually, at [f453990](https://github.com/unit8co/darts/pull/1557/commits/f4539901c257ef58d1c9e89ded46c2d90f1fcaaa) I fixed that, so I must've pushed it a bit after you pulled my previous changes